tests/LinqKit.Tests.Net452/PredicateBuilderTests.cs

```diff
@@ -14,6 +14,51 @@ private class User
             public int Id2 { get; set; }
         }
 
+        [Fact]
+        public void PredicateBuilder_Create()
+        {
+            var ienumerable = new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, }
```

Comment:
You could also use https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.range?view=net-5.0